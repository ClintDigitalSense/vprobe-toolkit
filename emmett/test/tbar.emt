GUEST:nt!NtCreateThread { foo = 1 * 32; ticks[RIP] <- foo; }
