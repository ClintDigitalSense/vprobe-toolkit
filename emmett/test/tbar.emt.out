(version 0.2)
(definteger foo)
(defaggr ticks 1 0)

 ;;;;;
(vprobe GUEST:nt!NtCreateThread ( do  ( setint foo   32)( aggr  ticks ( RIP) () foo)))